packages/fether-react/src/Accounts/CreateAccount/AccountImportOptions/AccountImportOptions.js

```diff
@@ -98,13 +135,22 @@ class AccountImportOptions extends Component {
       return;
     }
 
-    const chainId = parseInt(chainIdString);
+    const signerChainId = parseInt(chainIdString);
+
+    if (!this.isCurrentChainIdTheAddressForImportChainId(signerChainId)) {
+      this.setState({
+        error: `Parity Signer account chainId ${chainIdString} (${
```

Comment:
Users don't need to learn yet another term (chain id) or know what chain id Kovan is :)
I'd be more generic and talk about networks: 
Network mismatch, please import a Parity Signer account for the XYZ Network.

ps: note that `foundation` translates to `Ethereum mainnet` outside of Parity.

Reply:
Agreed. Although when the wording becomes longer the error message displays across 3 rows instead of 2 rows, such that the user has to scroll to read it. I assume we'll address this in a the separate PR that's dedicated to notifications https://github.com/paritytech/fether/issues/331 ?

Reply:
yes, notification will be handled in their own PR for sure.

Reply:
I've pushed a commit 1c24567d2642c3f4e72e5384a40a957df584646f that addresses this